src/main/java/io/epirus/console/project/NewProjectCommand.java

```diff
@@ -48,59 +37,10 @@
         optionListHeading = "%nOptions:%n",
         footerHeading = "%n",
         footer = "Epirus CLI is licensed under the Apache License 2.0")
-public class NewProjectCommand implements Runnable {
+public class NewProjectCommand extends ProjectOptions implements Runnable {
 
-    @CommandLine.ArgGroup() ProjectType projectType = new ProjectType();
```

Comment:
Please import `CommandLine.Option`, `CommandLine. Parameters ` and `CommandLine.ArgGroup`.